### PR TITLE
Query-frontend: use detailed text description for cmd

### PR DESCRIPTION
**Pull request by MalloZup**
# Description:

this is a minor PR fix I just fund out when doing some exploratory usage of thanos.

The text is copied from upstream documentation, feel free to suggest changes if you think it could be improved. :sake: 

## Changes

- add more text description for the command

#### Before:
```
  query-frontend [<flags>]
    query frontend
```

#### After:
```
  query-frontend [<flags>]
    query frontend command implements a service that can be put in front of Thanos Queriers to improve the read path. It is based on the Cortex Query Frontend component so you can find some common features like Splitting and Results
    Caching. Query Frontend is fully stateless and horizontally scalable.
```

### CODE REVIEW

**Comment by MalloZup** [comment]
@bwplotka or @yeya24  once you have 1 sec would be awesome if yuo just review it.. because I fear I will forget about this.. :grin:  happy new year

File: cmd/thanos/query_frontend.go [hunk 1]
```diff
@@ -43,7 +43,9 @@ type queryFrontendConfig struct {
 
 func registerQueryFrontend(app *extkingpin.App) {
 	comp := component.QueryFrontend
-	cmd := app.Command(comp.String(), "query frontend")
+	cmd := app.Command(comp.String(), "query frontend command implements a service that can be put in front of Thanos Queriers"+
+		" to improve the read path. It is based on the Cortex Query Frontend component so you can find"+
+		" some common features like Splitting and Results Caching. Query Frontend is fully stateless and horizontally scalable.")
```
Comment (yeya24) [inline cmd/thanos/query_frontend.go:48]: The description should be more concise. What about `a service deployed in front of queriers to improve query parallelisation and caching` ?

Comment (MalloZup) [reply cmd/thanos/query_frontend.go]: ok thx! I update it now

File: docs/components/query-frontend.md [hunk 2]
```diff
@@ -119,7 +119,8 @@ Naming is hard :) Please check [here](https://github.com/thanos-io/thanos/pull/2
 ```$
 usage: thanos query-frontend [<flags>]
 
-query frontend
+Query frontend command implements a service deployed in front of queriers to improve query parallelisation and caching
```
Comment (bwplotka) [inline docs/components/query-frontend.md:122]:
```suggestion
Query frontend command implements a service deployed in front of queriers to improve query parallelisation and caching.
```

**Comment by MalloZup** [comment]
@bwplotka thx..  will rebase

File: cmd/thanos/query_frontend.go [hunk 3]
```diff
@@ -43,7 +43,8 @@ type queryFrontendConfig struct {
 
 func registerQueryFrontend(app *extkingpin.App) {
 	comp := component.QueryFrontend
-	cmd := app.Command(comp.String(), "query frontend")
+	cmd := app.Command(comp.String(), "query frontend command implements a service deployed in front of queriers to improve query parallelisation and caching"+
```
Comment (yeya24) [inline cmd/thanos/query_frontend.go:46]: Should be `parallelization` IMO?

File: docs/components/query-frontend.md [hunk 4]
```diff
@@ -119,7 +119,9 @@ Naming is hard :) Please check [here](https://github.com/thanos-io/thanos/pull/2
 ```$
 usage: thanos query-frontend [<flags>]
 
-query frontend
+query frontend command implements a service deployed in front of queriers to
+improve query parallelisation and cachingQuery Frontend is fully stateless and
```
Comment (yeya24) [inline docs/components/query-frontend.md:123]: ditto

File: cmd/thanos/query_frontend.go (hunk 3 above)
Comment (yeya24) [reply cmd/thanos/query_frontend.go:46]: It is not necessary to add `Query Frontend is fully stateless and horizontally scalable.` to the description. First sentence is fine.

Comment (MalloZup) [reply cmd/thanos/query_frontend.go:46]: https://en.wiktionary.org/wiki/parallelisation bot are valid but I will use the more common `parallelization`

**Review by yeya24 — APPROVED** [review]
Thanks for your contribution! Will merge on green.

**Comment by MalloZup** [comment]
thx @yeya24 for patience :sun_with_face: \

**Pull request merged**